code/game/objects/items.dm

```diff
@@ -50,71 +53,103 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
+	var/slowdown = 0
+	///percentage of armour effectiveness to remove
+	var/armour_penetration = 0
+	///suit storage stuff.
+	var/list/allowed = null
+	///In deciseconds, how long an item takes to equip; counts only for normal clothing slots, not pockets etc.
+	var/equip_delay_self = 0
+	///In deciseconds, how long an item takes to put on another person
+	var/equip_delay_other = 20
+	///In deciseconds, how long an item takes to remove from another person
+	var/strip_delay = 40
+	///How long it takes to resist out of the item (cuffs and such)
 	var/breakouttime = 0
 
-	var/list/attack_verb //Used in attackby() to say how something was attacked "[x] has been [z.attack_verb] by [y] with [z]"
-	var/list/species_exception = null	// list() of species types, if a species cannot put items in a certain slot, but species type is in list, it will be able to wear that item
+	///Used in attackby() to say how something was attacked "[x] has been [z.attack_verb] by [y] with [z]"
+	var/list/attack_verb
+	///list() of species types, if a species cannot put items in a certain slot, but species type is in list, it will be able to wear that item
+	var/list/species_exception = null
 
+	///Who threw the item
 	var/mob/thrownby = null
 
-	mouse_drag_pointer = MOUSE_ACTIVE_POINTER //the icon to indicate this object is being dragged
+	///the icon to indicate this object is being dragged
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 
+	///Does it embed and if yes, what kind of embed
 	var/list/embedding = NONE
 
-	var/flags_cover = 0 //for flags such as GLASSESCOVERSEYES
+	///for flags such as GLASSESCOVERSEYES
+	var/flags_cover = 0
 	var/heat = 0
 	///All items with sharpness of IS_SHARP or higher will automatically get the butchering component.
 	var/sharpness = IS_BLUNT
 
+	///The type of tool, engineering or mining for example
 	var/tool_behaviour = NONE
+	///How fast does the tool work
 	var/toolspeed = 1
 
 	var/block_chance = 0
 	var/hit_reaction_chance = 0 //If you want to have something unrelated to blocking/armour piercing etc. Maybe not needed, but trying to think ahead/allow more freedom
-	var/reach = 1 //In tiles, how far this weapon can reach; 1 for adjacent, which is default
+	///In tiles, how far this weapon can reach; 1 for adjacent, which is default
+	var/reach = 1
 
-	//The list of slots by priority. equip_to_appropriate_slot() uses this list. Doesn't matter if a mob type doesn't have a slot.
-	var/list/slot_equipment_priority = null // for default list, see /mob/proc/equip_to_appropriate_slot()
+	///The list of slots by priority. equip_to_appropriate_slot() uses this list. Doesn't matter if a mob type doesn't have a slot. For default list, see [/mob/proc/equip_to_appropriate_slot()]
+	var/list/slot_equipment_priority = null
 
-	// Needs to be in /obj/item because corgis can wear a lot of
-	// non-clothing items
+	/// Wheher dogs can wear the item: Needs to be in /obj/item because corgis can wear a lot of non-clothing items

```

Comment:
not if they can wear the item, but instead a reference to the datum that determines behaviour if they do.